pkg/controller/federatedcluster/clusterclient.go

```diff
@@ -40,10 +40,13 @@ const (
 	LabelZoneRegion        = "failure-domain.beta.kubernetes.io/region"
 )
 
+// ClusterClient provides a method to compute the FederatedClusterStatus of a
+// particular cluster.
 type ClusterClient struct {
 	kubeClient *kubeclientset.Clientset
 }
 
+// NewClusterClientSet returns a ClusterClient for the given FederatedCluster.
```

Comment:
What is the point of adding comments that are obvious, as this one is?

Reply:
I had intended to add some comment indicating how the parameters were used - just apparently hadn't gotten to it before I posted the PR.